frontend/bot/src/Coda.re

```diff
@@ -24,3 +24,48 @@ let sendPayment = (~from, ~to_, ~amount, ~fee) => {
     (),
   );
 };
+
+module Unlock = [%graphql
+  {| mutation unlock($password: String, $publicKey: PublicKey) {
+      unlockWallet(input: {password: $password, publicKey: $publicKey}) {
+          publicKey
+        }
+  } |}
+];
+
+let unlock = (~publicKey, ~password) => {
+  let log = (w, fmt) => Logger.log("Unlock", w, fmt);
+
+  ReasonUrql.Client.executeMutation(
+    ~client=Graphql.client,
+    ~request=
+      Unlock.make(
+        ~password,
+        ~publicKey=Graphql.Encoders.publicKey(publicKey),
+        (),
+      ),
+    (),
+  )
+  |> Wonka.forEach((. {ReasonUrql.Client.Types.response}) =>
+       switch (response) {
+       | Data(_) => log(`Info, "Unlock successful of %s", publicKey)
+       | Error(e) =>
+         log(
+           `Error,
+           "Unlock failed for %s, error: %s",
```

Comment:
Maybe can newline before the error? Public keys are so long right now, that this will be kinda unreadable

Reply:
I was keeping all the logs on one line so that it would be easier to grep for them